install/kubernetes/cilium/values.yaml.tmpl

```
@@ -854,6 +854,13 @@ gatewayAPI:
   # Note that this value will be ignored when `hostNetwork.enabled == true`.
   # ref: https://kubernetes.io/docs/reference/networking/virtual-ips/#external-traffic-policy
   externalTrafficPolicy: Cluster
+
+  gatewayClass:
+    # -- Enable creation of GatewayClass resource
+    # default value is 'auto' which decides according to presence of gateway.networking.k8s.io/v1/GatewayClass in the cluster
+    # other possible values are 'true' and 'false'
```

```suggestion
    # The default value is 'auto' which decides according to presence of gateway.networking.k8s.io/v1/GatewayClass in the cluster.
    # Other possible values are 'true' and 'false', which will either always or never create the GatewayClass, respectively.
```
Small edit for grammar here, which should make this render out more nicely in the other files.